Problem running `make codegenTest`

I am looking to generate the example GOOL programs under `drasil-code/test`. Based on the `Makefile`, I tried `make codegenTest`. However, this gives me the following error.
```
make: *** No rule to make target `codegenTest_gen', needed by `codegenTest'.  Stop.
```
I tried adding `codegenTest` to `EXAMPLES` in the `Makefile` just to test it out, but then `make codegenTest` results in the following error.
```
stack build --ghc-options="-Wall -j2" ""
cd "build/" && stack exec -- "" 
Executable named  not found on path: ["/Users/hrz/Drasil/code/.stack-work/install/aarch64-osx/4bbcd6936a38b56e3f3683c66cbd1584abde75323f410df1341259bc5a6ea9c9/9.2.7/bin","/Users/hrz/.stack/snapshots/aarch64-osx/4bbcd6936a38b56e3f3683c66cbd1584abde75323f410df1341259bc5a6ea9c9/9.2.7/bin","/Users/hrz/.stack/compiler-tools/aarch64-osx/ghc-9.2.7/bin","/Users/hrz/.stack/programs/aarch64-osx/ghc-9.2.7/bin","/Users/hrz/.opam/default/bin","/Users/hrz/.nix-profile/bin","/nix/var/nix/profiles/default/bin","/opt/homebrew/bin","/opt/homebrew/sbin","/usr/local/bin","/System/Cryptexes/App/usr/bin","/usr/bin","/bin","/usr/sbin","/sbin","/Library/TeX/texbin","/Library/Apple/usr/bin","/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin","/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin","/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin","/opt/homebrew/anaconda3/bin","/Users/hrz/bin","/usr/local/plan9/bin","/Users/hrz/.cabal/bin","/opt/homebrew/opt/llvm@12/bin"]
make: *** [codegenTest_gen] Error 1
```
Is there some other method that I am supposed to use to generate the GOOL examples?

--- a/code/Makefile
+++ b/code/Makefile
@@ -269,6 +269,12 @@ $(filter %$(GEN_E_SUFFIX), $(GEN_EXAMPLES)): %$(GEN_E_SUFFIX):
 	@mkdir -p "$(BUILD_FOLDER)$(EDIR)"
 	cd "$(BUILD_FOLDER)$(EDIR)" && stack exec -- "$(EEXE)" $(EXECARGS)
 
+# Compiles GOOL examples to concrete code (Java, C++, etc.)
+$(GOOLTEST)$(GEN_E_SUFFIX):
+	stack build $(stackArgs) "drasil-code:exe:$(GOOLTEST_EXE)"
+	@mkdir -p "$(BUILD_FOLDER)$(GOOLTEST_DIR)"
+	cd "$(BUILD_FOLDER)$(GOOLTEST_DIR)" && stack exec -- "$(GOOLTEST_EXE)" $(EXECARGS)
+
 # Install individual Drasil examples
 $(filter %$(INSTALL_E_SUFFIX), $(INSTALL_EXAMPLES)): %$(INSTALL_E_SUFFIX):
 	stack install $(stackArgs) "$(EEXE)"

--- a/code/drasil-code/package.yaml
+++ b/code/drasil-code/package.yaml
@@ -89,7 +89,7 @@ library:
   - condition: false
     other-modules: Paths_drasil_code
 
-tests:
+executables:
   codegenTest:
     main: Main
     source-dirs: test

--- a/code/hie.yaml
+++ b/code/hie.yaml
@@ -7,7 +7,7 @@ cradle:
       component: "drasil-code:lib"
 
     - path: "drasil-code/test"
-      component: "drasil-code:test:codegenTest"
+      component: "drasil-code:exe:codegenTest"
 
     - path: "drasil-data/lib"
       component: "drasil-data:lib"

